### PR TITLE
Thebranch

### PR DESCRIPTION
**Pull request by collinthedavis**
The branch is ready.

### CODE REVIEW

**Comment by Anthony7147** [comment]
good job have some chocolate

File: README.rdoc [hunk 1]
```diff
@@ -1,7 +1,11 @@
+Once upon a time there was a dog that loved eating chocolate.
+However too much chocolate can kill a dog so he lives in misery knowing that doing the thing he loves could kill him, so he refrains.
```
Comment (silvercoulson) [inline README.rdoc]: That is so depressing.

**Comment by cmckee-dev** [comment]
Youre a Towell !!
https://giphy.com/gifs/south-park-jDcS7aNTwwMso

**Comment by nicoletingey** [comment]
ya eat some chocolate

**Comment by jasonamclean** [comment]
chocolate kills everything!

**Pull request merged**